src/spryker/application/skeleton/composer-harness.json

```diff
@@ -3,7 +3,8 @@
   "description": "Startup template for Spryker projects.",
   "type": "project",
   "require": {
-    "symfony/console": "^3.4.30"
+    "spryker/web-profiler": "^1.6",
```

Comment:
Needed due to references in the non-development codebase to the profiler bundle.